src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -121,6 +133,21 @@ public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> conf
                 throw new ArgumentException($"Invalid value '{basicAuthSource}' specified for property '{SchemaRegistryConfig.PropertyNames.SchemaRegistryBasicAuthCredentialsSource}'");
             }
 
+            var keySubjectNameStrategyString = Convert.ToString(config.FirstOrDefault(prop => prop.Key.ToLower() == SchemaRegistryConfig.PropertyNames.SchemaRegistryKeySubjectNameStrategy).Value) ?? "";
```

Comment:
config is a IEnumerable<KeyValuePair<string, string>>. I don't think this Convert.ToString is needed anymore.

Reply:
removed elsewhere as well.